packages/evm/src/precompiles/index.ts

```diff
@@ -138,6 +147,88 @@ const precompileEntries: PrecompileEntry[] = [
     precompile: precompile0a,
     name: 'KZG (0x0a)',
   },
+  // 0x00..0b: beacon block root, see PR 2810
```

Comment:
Where did the beacon block root precompile go to?

Reply:
Good question :thinking:  Will have to look